api-search/src/main/groovy/org/cedar/onestop/api/search/service/LogstashETLService.groovy

```diff
@@ -0,0 +1,19 @@
+package org.cedar.onestop.api.search.service
+
+import org.springframework.stereotype.Service
+
+@Service
+class LogstashETLService {
+
+    Map etlResponse(Map esResponse) {
```

Comment:
This is more a helper function than an ETL, and it's purpose isn't particularly clear once we lose the context that it was for aggregate results for recent search terms. What it actually does is produce a map of terms and number of occurrences, so we should name it accordingly.